<launch>
	<!--<include file="$(find panda_moveit_config)/launch/arm_warnings_test.launch"/> !-->

	<node name="singularity_marker" pkg="arm_warnings" type="singularity_marker">
		<rosparam command="load" file="$(find arm_warnings)/config/panda_sim.yaml"/>
	</node>
</launch>